### DISCUSSION TITLE
Snippet: How to register a block variation but hide it from the inserter

### DISCUSSION THREAD

**Discussion by ryanwelcher** [WordPress/developer-blog-content#348] [Topic Ideas]
In the [#outreach channel](https://wordpress.slack.com/archives/C015GUFFC00/p1732289265960489), I had a discussion with someone that wanted to create a block variation but only have it be inserted via a pattern.

This is possible by using the `scope` property and settings it to `block'

**Comment by bph** [comment]
Snippet approved. Conversation continue on this issue: 
- #351

**Comment by ryanwelcher** [comment]
This was published - https://developer.wordpress.org/news/snippets/snippet-how-to-register-a-block-variation-but-hide-it-from-the-inserter/
